shared/src/commonMain/kotlin/io.projectnewm.shared/login/models/LoginResponse.kt

```diff
@@ -0,0 +1,35 @@
+package io.projectnewm.shared.login.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class LoginResponse(
```

Comment:
Does LoginResponse need to be returned by LoginUseCase?  I thought accessToken and refreshToken would be stored in KMM.  I don't think the clients would need to access them directly eh?